torch_xla/__init__.py

```diff
@@ -116,11 +116,27 @@ def _summarize_fn_tracker():
 
 def _aws_ec2_inf_trn_init():
   try:
-    from torch_neuronx import xla
+    from libneuronxla.libneuronpjrt_path import libneuronpjrt_path
   except ImportError:
-    return
+    pass
   else:
-    xla.init()
+    os.environ.setdefault('PJRT_DEVICE', 'NEURON')
+    os.environ.setdefault('NEURON_LIBRARY_PATH', libneuronpjrt_path())
+    # enable addition features and overrides
+    try:
+      from torch_neuronx import xla
+    except ImportError:
+      pass
+    else:
+      xla.init()
+
+    # basic initializations if torch-neuronx is not available
+    from ._internal import neuron
+    if os.path.basename(sys.argv[0]) != 'neuron_parallel_compile':
```

Comment:
Does this make sense in `configure_multiprocess`?

https://github.com/pytorch/xla/blob/f45775a5aeafb5219a045ab96d240b89c4106f42/torch_xla/experimental/plugins.py#L36-L43

If not, that's fine. After the release, I can add a new plugin method to be called during init (e.g. `on_register`).

Reply:
For torchrun, this location is required. Unless you can show me another way.